### DISCUSSION TITLE
Feature request: Paperless-ngx container

### DISCUSSION THREAD

**Discussion by mrwsl** [tteck/Proxmox#456] [Ideas]
First of all thanks for your scripts! They made my switch to proxmox a procedure of one hour :)

It would be a nice addition to have a setup script for a Paperless ngx containter. Thanks!

**Comment by tteck** [comment]
Done!

**Reply by mrwsl** [reply]
Just to be clear: updating needs to be done manually, correct?

**Reply by tteck** [reply]
Yes, https://paperless-ngx.readthedocs.io/en/latest/administration.html#bare-metal-route
I'm working on an update script
